### ISSUE TITLE
[TTS]没有设置参数但使用了该参数的值

### ISSUE THREAD

**Issue by AlphaMind123** [PaddlePaddle/PaddleSpeech#3729]
在vits模型文件下的wavenet 文件中的residual_block.py文件中的第127行的下面代码中，
x = F.dropout(x, p=self.dropout_rate, training=self.training)
参数training使用的self.training,没有在该py文件的任何地方定义，导致报错。
![image](https://github.com/PaddlePaddle/PaddleSpeech/assets/108966996/39a02136-c8cb-4796-aa3d-e2ec8eca1cf2)

**Comment by Ray961123** [comment]
开发者你好，感谢关注 PaddleSpeech 开源项目，抱歉给你带来了不好的开发体验，目前开源项目维护人力有限，你可以尝试通过修改 PaddleSpeech 源码的方式自己解决，或请求开源社区其他开发者的协助。飞桨开源社区交流频道：[飞桨AI Studio星河社区-人工智能学习与实训社区](https://aistudio.baidu.com/community/channel/306)
